comp/updater/telemetry/telemetryimpl/telemetry.go

```diff
@@ -32,7 +33,13 @@ func Module() fxutil.Module {
 
 func newTelemetry(deps dependencies) (telemetry.Component, error) {
 	env := env.FromConfig(deps.Config)
-	telemetry, err := fleettelemetry.NewTelemetry(env.APIKey, env.Site, "datadog-installer")
+	telemetry, err := fleettelemetry.NewTelemetry(env.APIKey, env.Site, "datadog-installer",
+		fleettelemetry.WithSamplingRules(
+			tracer.NameServiceRule("cdn.*", "datadog-installer", 0.1),
```

Comment:
are those actually effective since this doesn't pass through the agent?
you might need to enable dropping of P0s too

Reply:
here's the config: https://github.com/DataDog/dd-trace-go/commit/30bfd5205d6af7bc8928b90e92310ae35879726b

Reply:
The tracing lib computes the sampling rate using these rules and adds the `_dd.rule_psr` tag that we read. The debug log I added shows that our client to the telemetry intake endpoint only sends traces that pass the sampling.

Reply:
thanks, yes I just realised that you added an extra sampler not in the tracer reading the rest